src/vm/moar/QAST/QASTCompilerMAST.nqp

```diff
@@ -2761,6 +2775,10 @@ class MoarVM::BytecodeWriter {
         self.align;
         self.write_extops;
         self.align;
+
+        nqp::die("Cannot have more than 65534 MAST::Frame in a CompUnit. This CU has " ~ nqp::elems(@!frames))
```

Comment:
Why the differences (65535 vs 65534 and 0xffff vs 65535) from the previous checks?

Reply:
thanks, i unified the messages